misc/swagger-ballerina/modules/swagger-to-ballerina-generator/src/main/java/org/ballerinalang/swagger/model/BalOpenApi.java

```diff
@@ -35,41 +39,100 @@
  * Wrapper for <code>io.swagger.oas.models.OpenAPI</code>
  * <p>This class can be used to push additional context variables for handlebars</p>
  */
-public class OpenApiWrapper {
-
+public class BalOpenApi {
```

Comment:
I feel OpenAPIWrapper is more suitable for this. Any thoughts?

Reply:
This model class is moving bit away from just being a wrapper for OpenAPI type. Shall we use **BallerinaOpenApi**?

Reply:
+1